src/k8s-engine/k3sHelper.ts

```diff
@@ -324,7 +324,18 @@ export default class K3sHelper extends events.EventEmitter {
       await this.waitForNetwork();
       await this.readCache();
       console.log(`Updating release version cache with ${ Object.keys(this.versions).length } items in cache`);
-      const channelResponse = await fetch(this.channelApiUrl, { headers: { Accept: this.channelApiAccept } });
+      let channelResponse: Response;
```

Comment:
Alternatively:
```suggestion
      let channelResponse: Awaited<ReturnType<fetch>>;
```
(All this does is save an import, really.)

Reply:
I prefer having the more explicit code when we give the actual name of the type, rather than reference the type of a value.